packages/website/pages/new-file.js

```diff
@@ -176,6 +182,25 @@ export default function NewFile() {
             </div>
           </form>
         </Box>
+        <When condition={error !== ''}>
+          <Alert
+            className="pa4 white"
+            position="top"
+            type="error"
+            timer={4}
+            onTimerEnd={() => setError('')}
+          >
+            <>
+              {error}{' '}
+              <button
+                className="border ml2 br-100 b--transparent pointer"
+                onClick={() => setError('')}
+              >
+                X
```

Comment:
Would be good if this were a proper cross graphic and not an X...🙏

Reply:
We already had a cross icon 😅